src/main/archetype/ui.content/src/main/content/jcr_root/content/dam/formsanddocuments-themes/__appId__/fsi/.content.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:fd="http://www.adobe.com/aemfd/fd/1.0" xmlns:dam="http://www.day.com/dam/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="dam:Asset">
+    <jcr:content
+        cq:conf=""
+        jcr:lastModified="{Date}2022-09-01T18:48:10.103-07:00"
+        jcr:primaryType="dam:AssetContent"
```

Comment:
only assets and renditions files are present in the theme, aren't there any other files in theme?

Reply:
This is AF1 theme. Theme definition is at [/fsi/_jcr_content/renditions/theme-json/.content.xml](https://github.com/adobe/aem-project-archetype/pull/1039/files#diff-35d4c4f2704b2965011c960dfa38c389a59c153b3b9f41ca58ff0cc2561bb637) When imported to the system, clientlibrary is generated by a listener.